### PR TITLE
Dev kmr

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: .github/actions/common-code/common-code.sh [hunk 1]
```diff
@@ -73,3 +73,82 @@ function validate_ci_endpoint {
...
+        echo "captured_output:"
+        echo ">>>>>"
+        echo "${captured_output}"
+        echo "<<<<<"
+
+        log_rc ${rc}
+        echo
+        echo
+
+        if [ ${RC_STATUS} == 0 ]; then
```
Comment (rafaelfolco) [inline .github/actions/common-code/common-code.sh:139]: RC_STATUS is 0, I am sure. Trust me.

File: .github/workflows/crucible-ci.yaml [hunk 2]
```diff
@@ -224,15 +228,14 @@ jobs:
         scenarios: "${{ matrix.benchmark }}"
         userenvs: "${{ matrix.userenv }}"
         ci_param_mode: "${{ matrix.param_mode }}"
-        run_samples: 1
-        repeat_runs: "no"
```
Comment (rafaelfolco) [inline .github/workflows/crucible-ci.yaml:202]: should we replicate this to all jobs ?

**Review by rafaelfolco — CHANGES_REQUESTED** [review]

File: .github/workflows/crucible-ci.yaml (hunk 2 above)
Comment (k-rister) [reply .github/workflows/crucible-ci.yaml:202]: Yes, that is the plan.  After finishing this PR I was going to work on rolling out some updates like this to other repos.

File: .github/actions/common-code/common-code.sh (hunk 1 above)
Comment (k-rister) [reply .github/actions/common-code/common-code.sh:139]: lol, I believe you are right...I have to think that the code has slightly changed over time to include this redundancy.  That said, can we defer fixing this to another PR?

Comment (rafaelfolco) [reply .github/actions/common-code/common-code.sh:139]: I am ok to merge as is

File: .github/workflows/crucible-ci.yaml (hunk 2 above)
Comment (rafaelfolco) [reply .github/workflows/crucible-ci.yaml:202]: ack

**Review by rafaelfolco — APPROVED** [review]

**Pull request merged**